scripts/import_default_vpc.py

```diff
@@ -0,0 +1,77 @@
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    cached_property,
+    config,
+    logging,
+)
+from azul.deployment import (
+    aws,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+class ImportDefaultVPC:
+
+    def main(self) -> None:
+        if config.terraform_component == 'shared':
+            self.import_default_vpc()
+        else:
+            log.info("The 'shared' component is not selected, skipping import.")
+
+    @cached_property
+    def default_vpc_id(self) -> str:
+        filters = [
+            {
+                'Name': 'isDefault',
+                'Values': ['true']
+            }
+        ]
+        vpcs = aws.ec2.describe_vpcs(Filters=filters)
+        return one(vpcs['Vpcs'])['VpcId']
+
+    def default_security_group_id(self, vpc_id: str) -> str:
+        filters = [
+            {
+                'Name': 'vpc-id',
+                'Values': [vpc_id]
+            },
+            {
+                'Name': 'group-name',
+                'Values': ['default']
+            }
+        ]
+        groups = aws.ec2.describe_security_groups(Filters=filters)
+        return one(groups['SecurityGroups'])['GroupId']
+
+    def import_default_vpc(self) -> None:
+        resources = terraform.run('state', 'list').splitlines()
+
+        resource = 'aws_default_vpc.default'
```

Comment:
This (the `.default` part) is coupled to the TF config in a brittle way. This is OK for the other script, which is soon to be discarded, but not for this one.